package.json

```diff
@@ -44,30 +42,29 @@
   },
   "dependencies": {
     "global": "^4.3.2",
-    "video.js": "^6 || ^7"
+    "video.js": "^6 || ^7 || ^8"
```

Comment:
Are we certain we don't want to remove v6 here?

Reply:
Discussed offline and decided this can be considered in another PR